google-cloud-scheduler/pom.xml

```diff
@@ -84,6 +83,18 @@
       <artifactId>grpc-google-cloud-scheduler-v1</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-pubsub</artifactId>
+      <version>1.102.0</version>
```

Comment:
Would it be possible to add this version into the `<properties>` tag?

Reply:
It's only use at one place so it's doesn't make sense to add into `<properties>` tag